app/lib/errors.rb

```diff
@@ -1,21 +1,23 @@
-class EmailTooLongError < StandardError
-end
+module Errors
```

Comment:
there's various references to things in this module that are outside check_your_answers_controller - don't we now need to go and update them all?

Reply:
Yep, I've done that now :)